index.js

```diff
@@ -0,0 +1,105 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+fs = require('fs');
+
+var octokit;
+
+const orgName = core.getInput('org-name');
+const authToken = process.env["GITHUB_TOKEN"];
+const path = core.getInput("file-path");
+const name = core.getInput("commit-user-name");
+const email = core.getInput("commit-user-email");
+const message = core.getInput("commit-msg")
+// Eliminate Owner name from repo;
+const repo = process.env["GITHUB_REPOSITORY"].split("/")[1]
+
```

Comment:
create a new var named `repo_owner` here.
```
const repo_owner= process.env["GITHUB_REPOSITORY"].split("/")[0];
```

Reply:
@UKnowWhoIm you missed this

Reply:
Nope i changed it to
`const [repo_owner, repo_name] = process.env["GITHUB_REPOSITORY"].split("/")`
Idk why this is still showing the old code, Checkout line 14 of `[index.js](https://github.com/UKnowWhoIm/members/blob/main/index.js)

Reply:
@gautamkrishnar